recipes-omnect/initrdscripts/omnect-os-initramfs/flash-mode-2

```diff
@@ -143,6 +181,13 @@ run_flash_mode_2() {
         p=""
     fi
 
+    setup_fifo || return 1
+    setup_network eth0 || return 1
+    vfat_grub_dd_workaround "${dest_blk}${p}1"
+    run_bmap_tool || return 1
+    vfat_grub_fsck_workaround
```

Comment:
if vfat_grub_xxx are really grub specific, shouldn't they run also only on a grub machine i.e., proteced by 'if [[ " ${machine_features} " =~ " grub " ]]; then ... fi' as in vfat_grub_dd_workaround?
at least everything in vfat_grub_fsck_workaround() is always run opposed to vfat_grub_dd_workaround(), and I would expect a uniform handling in both functions if invocation is done unconditionally.

Reply:
removed